### DISCUSSION TITLE
Welcome to shopify-webpack-dev-workflow Discussions!

### DISCUSSION THREAD

**Discussion by krjo** [krjo/shopify-webpack-dev-workflow#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by seanmpage** [comment]
Hey @krjo - nto sure how to get ahold of you put i posted a question here - https://github.com/krjo/shopify-webpack-dev-workflow/discussions/11
